tests/snapshots/tests.contrib.fastapi.test_fastapi.test_long_running_websocket_session.json

```diff
@@ -0,0 +1,463 @@
+[[
+  {
+    "name": "websocket.receive",
+    "service": "fastapi",
+    "resource": "websocket /ws",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "websocket",
+    "error": 0,
+    "meta": {
+      "_dd.base_service": "tests.contrib.fastapi",
+      "_dd.dm.resource": "websocket /ws",
+      "_dd.dm.service": "fastapi",
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "687e66c700000000",
+      "_dd.span_links": "[{\"trace_id\": \"687e66c30000000046c8f81dddb41fd1\", \"span_id\": \"f7ec6580db727c78\", \"attributes\": {\"dd.kind\": \"executed_by\"}}]",
+      "component": "fastapi",
+      "language": "python",
+      "runtime-id": "d90f622614ec4c0b86d96e8e3e1fbc26",
+      "span.kind": "consumer",
+      "websocket.duration.style": "blocking",
+      "websocket.message.type": "text"
+    },
+    "metrics": {
+      "_dd.dm.inherited": 1,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "process_id": 76013,
+      "websocket.message.frames": 1,
+      "websocket.message.length": 7
+    },
+    "duration": 1005235000,
+    "start": 1753114311766617000
```

Comment:
Do these tailing zeros mean the Python tracer (or Python itself) is also limited to millisecond-level precision? Node.js is similar.